definitions/sanitize-html_v1.x.x/flow_>=v0.23.x/sanitize-html_v1.x.x.js

```diff
@@ -0,0 +1,23 @@
+type SanitizeHtml$Frame = {
+  tag: string,
+  attribs: Object,
+  text: string,
+  tagPosition: number,
+}
+
+type SanitizeHtml$SanitizeOptions = {
+  allowedTags?: Array<string>|false,
+  allowedAttributes?: {[key: string]: Array<string>}[]|false,
+  transformTags?: {[key: string]: string|(tagName: string, attribs: Object) => { tagName: string, attribs: Object}},
+  exclusiveFilter?: (frame: SanitizeHtml$Frame) => bool,
+  textFilter?: (text: string) => string,
+  allowedClasses?: {[key: string]: Array<string>},
+  allowedSchemes?: Array<string>|{[key: string]: Array<string>},
+  nonTextTags?: Array<string>,
+  parser?: Object,
```

Comment:
Based on the readme for `sanitize-html`, it looks like you can type the options object as:

```javascript
$Shape<{
  xmlMode: bool,
  lowerCaseTags: bool,
  lowerCaseAttributeNames: bool,
  recognizeCDATA: bool,
  recognizeSelfClosing: bool,
}>
```